app/assets/javascripts/quby/answers/validation.js

```diff
@@ -189,6 +208,48 @@
     return !failed;
   };
 
+  function parsePartialDate(inputs) {
+    var value = function(placeholder) {
+      var val = inputs.filter("[placeholder=" + placeholder + "]").first().val();
+      if(val === undefined || val == "")
```

Comment:
Krijg je nu niet dat als je alleen de dag invult dat hij dan onjuist geldig is?
Volgens mij moet vall=="" een invalidDate gooien.